JacobF94/NSS-Frontend-Capstone
# progress on individual day edit module

#### Changes Made
1. Removed each individual module for days and instead implemented one module that conditionally renders
2.  The singular 'Day' module correctly load information for currently logged in user as well as their exercise information for the given day that has been loaded
3. 
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jf-dayFinish
    ```
2. Open a new Terminal tab (⌘T or Ctrl+Shift+T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
